<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureSettingWidget</class>
 <widget class="QWidget" name="PictureSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>456</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>When Displaying Slideshows:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBoxExpand">
       <property name="text">
        <string>Expand Small Images</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelExpand">
       <property name="pixmap">
        <pixmap resource="softprojector.qrc">:/icons/icons/ExpandSmall.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButtonFit">
       <property name="text">
        <string>Fit Images To Screen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFit">
       <property name="pixmap">
        <pixmap resource="softprojector.qrc">:/icons/icons/FitToScreen.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="radioButtonFitExpand">
       <property name="text">
        <string>Fit Images To Screen By Expanding</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFitExpand">
       <property name="pixmap">
        <pixmap resource="softprojector.qrc">:/icons/icons/FitToScreenByExpanding.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxResize">
     <property name="title">
      <string>Resize Large Images on Import</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>It is highly recommended to reduce large images on import. This will improve load, save and display time of slideshows.
We recommend to resize images to display screen size.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Bound Box:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxBoundAmount">
          <item>
           <property name="text">
            <string>800 x 800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1024 x 1024</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1280 x 1280</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1366 x 1366</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1440 x 1440</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1600 x 1600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1920 x 1920</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Custom</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditBound">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelBoundBy"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>90</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxExpand</tabstop>
  <tabstop>radioButtonFit</tabstop>
  <tabstop>radioButtonFitExpand</tabstop>
  <tabstop>groupBoxResize</tabstop>
  <tabstop>comboBoxBoundAmount</tabstop>
  <tabstop>lineEditBound</tabstop>
 </tabstops>
 <resources>
  <include location="softprojector.qrc"/>
 </resources>
 <connections/>
</ui>
